automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/Notebook.scala

```diff
@@ -92,8 +92,8 @@ object Notebook extends RestClient with LazyLogging {
     val path = contentsPath(googleProject, clusterName, contentPath) + (if (includeContent) "?content=1" else "")
     logger.info(s"Get notebook contents: GET /$path")
     val cookie = Cookie(HttpCookiePair("LeoToken", token.value))
-
-    handleContentItemResponse(parseResponse(getRequest(url + path, httpHeaders = List(cookie))))
+    val referer = Referer(Uri("https://leonardo.dsde-dev.broadinstitute.org"))
+    handleContentItemResponse(parseResponse(getRequest(url + path, httpHeaders = List(cookie, referer))))
```

Comment:
I think there are other direct proxy API calls in this class.. not sure how all are used but probably should update all of them with Referer.

Reply:
just updated all the API calls, thanks for pointing that out :)